blocks-touch/b-slider/b-slider.js

```diff
@@ -332,6 +330,11 @@
 
             e = e.originalEvent;
 
+            // каскадная подписка на move / up
+            this
+                .bindToWin( 'pointermove', this._onPointerMove )
```

Comment:
@mishaberezin what about this https://github.com/bem/bem-bl/pull/603/files#diff-5bad4884874bbc035f147266d49365d0L174 ?
or
Only all new changes must be done by Codestyle?